thinkhazard/templates/admin/index.jinja2

```diff
@@ -80,7 +79,20 @@
                 Import from Transifex
               </a>
             </p>
-            -->
+            <p>
+              Push data from ThinkHazard DB to transifex for translation.
+            </p>
+            <p class="text-right">
+              <a
+                class="btn btn-info"
+                href="{{request.route_url('admin_add_task', _query={'task': 'transifex_push'})}}"
+                {% if 'transifex_push' in running_keys %}
```

Comment:
We talked about this with Arnaud & thought it would be better to disable every task when there's one running. May can be done in this PR.

Reply:
Ah good point, I will adapt that.